packages/sdk/src/types/runtime/actor.ts

```diff
@@ -346,7 +305,7 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	 * @param colliderType Type of the collider to enable.
 	 * @param isTrigger Whether the collider is a trigger volume or not.
 	 * @param center The center of the collider, or default of the object if none is provided.
-	 * @param radius The radius of the collider, or default bounding if non is provided.
+	 * @param radius The radius of the collider.

```

Comment:
What is the default radius? Would be good to mention it in the doc comment.

Reply:
Changed the API so the radius is no longer optional. There was no reasonable default radius that I could come up with.

Reply:
We were supposed to be allowing Unity to provide the default for sizing did we not?

---
In reply to: [314923618](https://github.com/microsoft/mixed-reality-extension-sdk/pull/415#discussion_r314923618) [](ancestors = 314923618)

Reply:
Changed this back, and documented the defaults.

---
In reply to: [315737144](https://github.com/microsoft/mixed-reality-extension-sdk/pull/415#discussion_r315737144) [](ancestors = 315737144,314923618)